.changeset/unlucky-mice-sneeze.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏add changeset

Can you add the changeset for `cli` and `client` to have minor version bumps?